README.md

```diff
@@ -19,29 +19,34 @@ Version 4.0 Adds explicit support for OAS 3.0 and OAS 3.1 as separate implementa
 From Typescript you can consume it from the library:
 
 ```typescript
-import { OpenAPIObject, OpenApiBuilder } from "openapi3-ts"; 
+import lib from "openapi3-ts"; 
+
+// And then use: lib.oas31
 ```
 
-Or direclty from the sources:
+Or direclty from sources:
 
 ```typescript
-import { OpenAPIObject, OpenApiBuilder } from "openapi3-ts/src"; 
+import { OpenAPIObject } from "openapi3-ts/src/model/openapi31"; 
+import { OpenApiBuilder } from "openapi3-ts/src/dsl/openapi-builder31"; 
 ```
 
 From a JavaScript application you can import:
 
 ```javascript
-import * as openapi31 from 'openapi3-ts';
- 
+import lib from 'openapi3-ts';
+
+// And then use: lib.oas31 
 ```
 
 ### To use version 3.0 import
 
 From Typescript you can consume it from the library:
 
 ```typescript
-import { OpenAPIObject } from "openapi3-ts/model/openapi30"; 
-import { OpenApiBuilder } from "openapi3-ts/dsl/openapi-builder30";
+import { oas30 } from "openapi3-ts"; 
```

Comment:
this does not work for some reason

> TS2614: Module '"openapi3-ts"' has no exported member 'oas30'. Did you mean to use 'import oas30 from "openapi3-ts"' instead?

Reply:
maybe need to clear cache... please wait....

Reply:
Yes. Looks like a typo on my side.
Can you confirm this works for you?

```typescript
import oas30 from "openapi3-ts";
```

Reply:
this would import the whole object (default) giving it the name `oas30`. Similar to `import lib from "openapi3-ts"` approach.

![image](https://user-images.githubusercontent.com/13189514/228615267-2ba07223-3d87-41ab-8c81-ae49c627963a.png)

The object itself is OK.

I believe the export from index.ts should be done in a slighlty different way.

Reply:
I found the reason, see my suggestion below, @pjmolina